tests/src/lib.rs

```diff
@@ -0,0 +1,91 @@
+//! This is a goldenfile test for osm2streets. Each test case has a directory with:
+//!
+//! - `input.osm`, from the export tab of https://www.openstreetmap.org
+//! - `cfg.json`, defining the `driving_side` and describing the situation
+//! - `output.json`, a GeoJSON file showing the resulting road and intersection polygons
```

Comment:
Can we call these `test.json` (or `test.yml`), `input.osm`, `rawmap.geojson`? I want to add tests of different kinds that sit along side the RawMap GeoJSON ones, so lets name each output what it is.

I might go for `roads.dot` (then `corridors.dot`)...